decidim-elections/app/queries/decidim/votings/admin/ballot_style_by_voting_code.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module Admin
+      # A class used to find trustees by participatory space.
+      class BallotStyleByVotingCode < Rectify::Query
+        # Syntactic sugar to initialize the class and return the queried objects.
+        #
+        # voting - the voting of the Vallot Style
```

Comment:
```suggestion
        # voting - the voting of the Ballot Style
```
Not sure, what a Vallot Style is. 😄
Same in the next line.

Reply:
Gosh, speaking Spanish all the time I'm losing the ability to distinguish `b` from `v` 😂